ui/ducks/metamask/metamask.js

```diff
@@ -373,7 +373,7 @@ export function isEIP1559Network(state, networkClientId) {
   return (
     state.metamask.networksMetadata?.[
       networkClientId ?? selectedNetworkClientId
-    ].EIPS[1559] === true
+    ]?.EIPS[1559] === true
```

Comment:
Hmm. Was this done to fix a type error? It does look like it would be a type error, but it's unclear how this update could trigger it.

Reply:
Yes TypeError: Cannot read properties of undefined (reading 'EIPS')